docs/zh/kpanda/07UserGuide/Workloads/CreateDeploymentByImage.md

```diff
@@ -50,12 +52,13 @@
 
     按照以下输入信息后，点击`确认`。
 
-    - 容器名称：输入新建容器的名称。请输入 4 到 63 个字符的字符串，可以包含小写英文字母、数字和中划线（-），并以小写英文字母开头，小写英文字母或数字结尾。例如 deployment-01。
-    - 容器镜像：从镜像仓库选择的镜像名称，也支持手动输入镜像名称（名称需为镜像仓库中已有的镜像名，否则将无法获取）。例如 nginx。
-    - 更新策略：容器执行更新时，镜像拉取策略。开启后工作负载每次重启/升级均会重新拉取镜像，否则只会在节点上不存在同名同版本镜像时拉取镜像。默认为：总是拉取镜像。
+    - 容器名称：请注意名称最长63个字符，只能包含小写字母、数字及分隔符（“_”）,且必须以小写字母或数字开头及结尾。例如 nginx-01。
+    - 容器镜像：选择镜像仓库后输入镜像名称或镜像地址来选择镜像。请注意，镜像名称必须为镜像仓库中已有的镜像名，否则将无法获取）。例如 您可以选择`公共镜像仓库` 后 输入 “nginx” 来部署 nginx 镜像。
+    - 更新策略：当对容器执行更新时，[镜像拉取策略](https://kubernetes.io/zh-cn/docs/concepts/containers/images/#image-pull-policy)。默认不启用`总是拉取镜像策略`，即在工作负载每次重启/升级时拉取本地镜像,当镜像在本地不存在时将会从镜像仓库重新拉取。如果启用`总是拉取镜像策略`将默认从仓库重新拉取镜像。
     - 特权容器：默认情况下，容器不可以访问宿主机上的任何设备，开启特权容器后，容器即可访问宿主机上的所有设备，享有宿主机上的运行进程的所有权限。默认启用。
-    - CPU 配额：容器 CPU 资源的最低使用量和最高限度。申请：容器需要使用的最小 CPU 值。限制：允许容器使用的 CPU 最大值。建议设容器配额的最高限额，避免容器资源超额导致系统故障。默认为 0.25，0.25。
-    - 内存配额：容器内存资源的最低使用量和最高限度。申请：容器需要使用的最小内存值。限制：允许容器使用的内存最大值。建议设容器配额的最高限额，避免容器资源超额导致系统故障。默认为 512 MB，512 MB。
+    - CPU 配额：容器 CPU 资源的最低使用量和最高使用量。申请：容器需要使用的最小 CPU 用量。限制：允许容器使用的最大 CPU 用量。请根据需要为容器配置 CPU 配额，避免资源浪费和因容器资源超额导致系统故障。默认为 0.25，0.25。
+    - 内存配额：容器内存资源的最低使用量和最高使用量。申请：容器需要使用的最小内用量。限制：允许容器使用的内存最大用量。请根据需要为容器配置 CPU 配额，避免资源浪费和因容器资源超额导致系统故障。默认为 512 MB，512 MB。
+    - GPU 配额：为集群配置 GPU 使用配额，需要管理员预先在集群上的节点上安装 GPU 卡及驱动插件，并在**集群设置**上开启了 GPU 特性后，您才能为集群配置 GPU 配额。GPU 配额设置支持为容器设置独享整张 GPU 卡或部分 vGPUs。例如，您有一张 8 核心的 GPU 卡，您可以输入数字 `8` 让容器独享整长卡，也可以输入数字 `1` 为容器配置 1 核心的 vGPU。
```

Comment:
```suggestion
    - GPU 配额：为集群配置 GPU 使用配额，需要管理员预先在集群上的节点上安装 GPU 卡及驱动插件，并在 **集群设置** 上开启了 GPU 特性后，您才能为集群配置 GPU 配额。GPU 配额设置支持为容器设置独享整张 GPU 卡或部分 vGPUs。例如，您有一张 8 核心的 GPU 卡，您可以输入数字 `8` 让容器独享整长卡，也可以输入数字 `1` 为容器配置 1 核心的 vGPU。
```

Reply:
We need spaces to make this bold format work well in Mkdocs. Would you please get it updated?